Jenkinsfile

```diff
@@ -5,6 +5,9 @@ pipeline {
 
   options {
     timestamps()
+    /* we want to avoid running in parallel otherwise we can get gcloud crashed : database is locked
```

Comment:
is `gcloud crashed : database is locked` the error? If so, please surround by quotes. 

Also, use two lines of `//` instead of `/*`

Reply:
`We want to avoid running in parallel. When we have 2 build running on the same environment (.e.g oc 3.11) in parallel, we get the error "gcloud crashed : database is locked"`

@eladkug does this happen only in GKE or also in OCP? If so, state that in the comment above.

Reply:
👍